### ISSUE TITLE
Svg transform not working with 2.0.13

### ISSUE THREAD

**Issue by zerismo** [thednp/kute.js#104]
Tried using svg transform with the following code:

```
var tween3 = KUTE.to('#element', {
  svgTransform: {
    translate: [50,50],
  }
})
tween3.start();

```
The HTML is as follows:
```
  <div class="hometop-header">
    <div class="hometop-header-wrapper">
      <div class="heart-wrap">
        <svg class="heart" id="heart" viewBox="0 0 1920 3000" xmlns="http://www.w3.org/2000/svg">
          <path id="element" d="M819.183 84C603.326 84 394.11 298.645 296.628 392.697 194.353 491.375 17.873 540.649 17.873 779.3c0 299.624 206.546 755.983 440.117 650.075 159.903-72.505 232.096-126.243 320.318-347.805 68.246-171.396 265.365-243.877 265.365-530.448C1043.673 251.499 1118.56 84 819.183 84z" stroke="#f8566a" fill="transparent"></path>
        </svg>
      </div>
    </div>
  </div>

```

**Comment by thednp** [comment]
I'm currently working on KUTE.js next version, please commit your example to fiddle or something so I can test on the spot. 

Thanks

**Comment by thednp** [comment]
I just tested your example, cannot confirm.

**Comment by zerismo** [comment]
@thednp , this is the fiddle for the above issue. https://codepen.io/zerismo/pen/MWyodPa

**Comment by thednp** [comment]
@zerismo [this is the official build](https://github.com/thednp/kute.js/blob/master/src/index.js) now, it doesn't feature the SVGTransform component anymore, you need to create your own custom builds. Perhaps you can use the `index-extra.js` build or a custom variation of that, with exactly what you need.

**Comment by thednp** [comment]
@zerismo, just to be clear, there is a reason why this component is not featured with the official distribution build, here's a quick note from the [TransformMatrix](https://thednp.github.io/kute.js/svgTransform.html) demo page (something you should consider using):

>Most importantly we have the update function which implements the [DOMMatrix()](https://developer.mozilla.org/en-US/docs/Web/API/DOMMatrix) API for smooth animation at no performance cost, which is different from other libraries that use a ***webkitCSSMatrix*** polyfill and lose performance.

Hope this makes it super clear. Cheers :)

**Comment by simonhuwiler** [comment]
Sorry to reopen a closed issue, but I am currently facing the same problem:
What is the best way to move a group (`g`) without `svgTransform`, but in the coordinate system of the SVG?
`transform` in the `style`-attribute uses either pixels or percentage.

Example: Move the Group `test` to svg-position 10,10:
```html
  <svg width="100%", height="365px">
    <g id="test" transform="translate(100, 100)">
      <circle r="10" stroke="black" stroke-width="3" fill="red" />
    </g>
  </svg>
```

```javascript
const buttonClick = () => {
  KUTE.to(
    "#test",
      { svgTransform: { translate: [10, 10]}}, // this will not work, because "svgTransform" is not in the build
      {duration: 500}
  ).start()
}
```

Many thanks!

**Comment by thednp** [comment]
Hi @simonhuwiler and thanks for dropping by.

Nowadays all major browsers support the new standard transform functions as CSS properties, even for SVGs (EG: I just tested `#test { rotate: 25deg }` and it works), the SVGTransform plugin could be bundled into a custom build to serve for a really old app which requires polyfils to work, for today's modern apps makes little sense.

I've started some time ago a Typescript sourced KUTE.js version for the future, I want all plugins modernized and modularized for tree shaking, a much more simple update function and support for major frontend UI libraries or render engines. But the time is not my friend lately.

In your case, yes, a custom build should meet your needs.

Have a good one.
